index.rst

```diff
@@ -48,7 +48,353 @@
 
    **This technote is not yet published.**
 
-   Narrative summary of the LSST Science Platform capabilities that will be available during the remainder of Construction (FY19+) and, specifically, for initial AuxTel commissioning, main telescope commissioning with ComCam and with the full camera, and for any early data releases prior to the start of formal operations.
+Narrative summary of the Rubin Science Platform capabilities that will be available for prompt (near-real-time through few-day time scales) access to acquired data during the remainder of Construction (FY23-25) and, specifically, for AuxTel commissioning and main telescope commissioning.
+
+Primarily motivated as a plan for the development of capabilities for the delivery of Prompt Products data, providing "practice" for the operation of the RSP for PP data in the first year of operations.
+(Note that the Data Preview 0 activities are all essentially DRP-centric, and even DP1 does not provide for continuously updated data deliveries.)
+
+One end point of the data access environment discussed in this document is the operational data-rights-holder access provided for Prompt Products at the beginning of the formal survey.
+This will be summarized in a future expansion of RTN-011.
+
+May also overlap somewhat with content in RTN-011 and other documents on the content and data access for early data releases (e.g., DP1) prior to the start of formal operations.
+
+We will use the term "Prompt Products Preview" or "PP Preview" internally to this technote to refer to the set of capabilities and procedures it describes.
```

Comment:
I get the need for a name here but "Preview" worries me--I think it's inevitable it will be mentioned in semi-public fora, and the analogy to the Data Previews may set the wrong expectations.

Prompt Products Pathfinder?  Prototype?